components/Input/style.css

```diff
@@ -0,0 +1,68 @@
+:root {
+  --input-border: #8b8a8b;
+  --input-focus-h: 240;
+  --input-focus-s: 100%;
+  --input-focus-l: 42%;
+}
+.input-box {
+  color: var(--black-color);
+  gap: 0px !important;
```

Comment:
Try to avoid using !important, as you may find it difficult to resolve conflicts in future.

Reply:
I used important to override the styling. I used gap in my ui. Thanks btw.